examples/experimental/ProductionMonitoring/stream_table_api.ipynb

```diff
@@ -0,0 +1,173 @@
+    "\n",
+    "View data at: https://weave.wandb.ai/?exp=get%28%0A++++%22wandb-artifact%3A%2F%2F%2Fstacey%2Fmesa%2Fmy_stream_table%3Alatest%2Fobj%22%29%0A++.rows\n",
+    "\n",
+    "All log calls on a given StreamTable instance will append the given rows to that instance.\n",
+    "\n",
+    "In a notebook, the StreamTable variable on a line by itself will return a Weave Panel view of the StreamTable. The StreamTable will contain all the logged columns and their values, as well as a `timestamp` column indicating when the row was logged. By default, rows will be ordered by oldest first. You can modify a StreamTable Panel as you would a wandb.Table from the UI: sort, filter, group, etc.\n",
+    "\n",
+    "**Note: Column display order is currently non-deterministic** over repeated views of the Panel. If you would like to modify the order, open the StreamTable Panel in a new window as a Board and edit/save a Board from this seed panel. There are two options to achieve this:\n",
+    "* via the weave.wandb.ai/?exp=... URL\n",
+    "* via \"Open in new tab\" arrow button, revealed in the menu when you hover on the right side of a StreamTable panel displayed in the notebok)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b928c2e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# log data to the StreamTable as a dictionary or list of dictionaries\n",
+    "st.log({\"col_a\" : \"1\", \"col_b\" : \"17\", \"col_c\" : \"42\"})\n",
+    "\n",
+    "# show the StreamTable\n",
+    "st"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "83cec916",
+   "metadata": {},
+   "source": [
+    "## Step 3: Log more data & explore the results!\n",
+    "\n",
+    "Continue logging as much data as you like to any StreamTable instance. You can keep a reference to a given Python StreamTable object in your notebook session or script, and you can reconnect to the same StreamTable instance across multiple sessions/runs of your script via the StreamTable's unique name (e.g. `st = StreamTable(\"stacey/mesa/my_stream_table\")` ) and keep adding rows. Multiple/parallel processes writing to the same StreamTable are also supported—the server will use a queue to order any concurrent messages.\n",
+    "\n",
+    "If you save the StreamTable Panel as a Board, the Board will continue to update as you send more data to the same StreamTable instance.\n",
+    "\n",
+    "\n",
+    "### Important usage notes\n",
+    "\n",
+    "Some details to keep in mind for this early iteration of the StreamTables API:\n",
+    "\n",
+    "* **columns must match exactly in each log**: each dictionary passed to `.log()` must contain a key for every column in the StreamTable instance. Skipping a key, even if its value is null, or adding a new key/column name after the first log call, may break the StreamTable. \n",
```

Comment:
this sounds like a bug that we should file and remove from the notebook

Reply:
oops thank you, I removed these from the README and not from notebook!